### ISSUE TITLE
IOS remove app from tray (error)

### ISSUE THREAD

**Issue by kkhalilov** [NativeScript/ios#222]
On an empty nativescript-vue project, if you throw the application out of the tray, an error pops up (IOS simulator IOS version 15.2)

It happens without any intervention in the application (everything out of the box)

Error Formulating Crash Report:
dyld_process_snapshot_get_shared_cache failed

{
  "name": "testApp",
  "main": "app/app.js",
  "version": "1.0.0",
  "private": true,
  "dependencies": {
    "@nativescript/core": "~8.5.0",
    "@nativescript/theme": "~3.0.2",
    "nativescript-vue": "~2.9.3"
  },
  "devDependencies": {
    "@nativescript/android": "8.5.0",
    "@nativescript/ios": "8.5.2",
    "@nativescript/webpack": "~5.0.14",
    "nativescript-vue-template-compiler": "~2.9.3"
  }
}

Xcode 14.2 (14C18) 
Version of nativescript - 8.5.3

**Comment by rigor789** [comment]
> throw the application out of the tray

Can you explain what you mean by this? Perhaps a video would help.

**Comment by kkhalilov** [comment]
> > throw the application out of the tray
> 
> Can you explain what you mean by this? Perhaps a video would help.


https://github.com/NativeScript/ios/assets/79607010/7a4bd00b-a2a7-4089-949e-6885eace37f1

**Comment by rigor789** [comment]
I believe this is fixed by https://github.com/NativeScript/ios/commit/5a6c2ee5efa0c557c94ae56da0d3b3a31911d1b8

Which will soon be released, but should be available on [8.5.3-next.2023-07-28-5696348438](https://www.npmjs.com/package/@nativescript/ios/v/8.5.3-next.2023-07-28-5696348438) for example.

Also note, this usually wouldn't affect a production build - since it "silently" crashes when trying to dispose after exit.

**Comment by kkhalilov** [comment]
> I believe this is fixed by [5a6c2ee](https://github.com/NativeScript/ios/commit/5a6c2ee5efa0c557c94ae56da0d3b3a31911d1b8)
> 
> Which will soon be released, but should be available on [8.5.3-next.2023-07-28-5696348438](https://www.npmjs.com/package/@nativescript/ios/v/8.5.3-next.2023-07-28-5696348438) for example.
> 
> Also note, this usually wouldn't affect a production build - since it "silently" crashes when trying to dispose after exit.

Yes you are right. When using the version [8.5.3-next.2023-07-28-5696348438](https://www.npmjs.com/package/@nativescript/ios/v/8.5.3-next.2023-07-28-5696348438), the problem is not reproduced. Thank you very much !
